SPECS/ldns/ldns.spec

```diff
@@ -32,48 +32,39 @@
 
 Summary:        Low-level DNS(SEC) library with API
 Name:           ldns
-Version:        1.7.0
-Release:        32%{?dist}
-License:        BSD
+Version:        1.8.3
+Release:        1%{?dist}
+License:        BSD-3-Clause
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
+Distribution:   Azure Linux
 Group:          System Environment/Libraries
 Url:            http://www.nlnetlabs.nl/%{name}/
 Source0:        http://www.nlnetlabs.nl/downloads/%{name}/%{name}-%{version}.tar.gz
-Patch1:         ldns-1.7.0-multilib.patch
-Patch2:         ldns-1.7.0-parse-limit.patch
-Patch3:         ldns-1.7.0-realloc.patch
-Patch4:         ldns-1.7.0-Update-for-SWIG-4.patch
 
-# Only needed for builds from svn snapshot
-%if 0%{snapshot}
-BuildRequires: libtool
 BuildRequires: autoconf
 BuildRequires: automake
-%endif
-
-BuildRequires: gcc, make
-BuildRequires: libpcap-devel
-%if %{with dane_ta}
-BuildRequires: openssl-devel >= 1.1.0
-%else
-BuildRequires: openssl-devel >= 1.0.2k
-%endif
-BuildRequires: gcc-c++
 BuildRequires: doxygen
-
-# for snapshots only
-# BuildRequires: libtool, autoconf, automake
-%if %{with python3}
-BuildRequires: python3-devel, swig
-%endif
+BuildRequires: gcc
+BuildRequires: gcc-c++
+BuildRequires: libpcap-devel
+BuildRequires: libtool
+BuildRequires: make
+BuildRequires: openssl-devel
 %if %{with perl}
 BuildRequires: perl-devel
 BuildRequires: perl-ExtUtils-MakeMaker
 BuildRequires: perl-generators
 BuildRequires: perl(Devel::CheckLib)
 %endif
+%if %{with python3}
+BuildRequires: python3-devel
+BuildRequires: swig
+%endif
 Requires: ca-certificates
+Requires: glibc
+Requires: openssl
+
+Patch1: ldns-swig-4.2.patch
```

Comment:
Can you move the `Patch` tags below the `Source` ones? That's where we usually keep and expect them.

Reply:
done